Source/RuntimeAudioImporter/Private/RuntimeAudioImporterLibrary.cpp

```diff
@@ -66,6 +67,10 @@ void URuntimeAudioImporterLibrary::ImportAudioFromFile(const FString& FilePath,
 	}
 
 	ImportAudioFromBuffer(MoveTemp(AudioBuffer), AudioFormat);
+#else
+	UE_LOG(LogRuntimeAudioImporter, Error, TEXT("File operation support disabled"));
```

Comment:
Could we add a more detailed description here? Such as: `UE_LOG(LogRuntimeAudioImporter, Error, TEXT("Unable to import audio from file '%s' because the file operation support is disabled"), *FilePath);`.